Add Jira Incident Layout V2 (#15825)
demisto/content: Packs/Jira/ReleaseNotes/1_4_1.md

#### Layouts
##### Jira Incident Layout

> - Removed "Jira Attachments" field and added **Attachments** section instead for solving an issue when attachments didn't appear in fetched incidents' layout.

Reviewer: ```suggestion
- Removed the *Jira Attachments* field and added the *Attachments* section instead to solve an issue where attachments wouldn't appear in a fetched incidents' layout.
```
